samcli/cli/cli_config_file.py

```diff
@@ -76,7 +75,6 @@ def __call__(self, config_path, config_env, cmd_names):
 
             # NOTE(TheSriram): change from tomlkit table type to normal dictionary,
             # so that click defaults work out of the box.
-            samconfig.sanity_check()
```

Comment:
Note: unnecessary as the `get_all()` below already validates the config by calling `_read()` (which is also what [`sanity_check()`](https://github.com/aws/aws-sam-cli/blob/3edb55cd8dba9898ddf2e1c8bd9eeba0434c08de/samcli/lib/config/samconfig.py#L134) does, except by swallowing the useful `tomlkit.exceptions.TOMLKitError`).